src/test/resources/features/comments.feature

```diff
@@ -32,3 +32,81 @@ Feature: Comments
     And I validate the response contains "text" equals "Comment updated by cucumber"
     And I send a DELETE request to "/projects/{P.id}"
     And I validate the response has status code 204
+
+  Scenario: GET comment
+    Given I send a "POST" request to "/projects" with json body
+    """
+    {
+    "name": "Project created by Cucumber"
+    }
+    """
+    And I save the response as "P"
+    And I send a "POST" request to "/projects/{P.id}/stories" with json body
+    """
+    {
+    "name": "Story created by Cucumber"
+    }
+    """
+    And I save the response as "S"
+    And I send a "POST" request to "/projects/{P.id}/stories/{S.id}/comments" with json body
+    """
+    {
+    "text": "Comment created by Cucumber"
+    }
+    """
+    And I save the response as "C"
+    When I send a GET request to "/projects/{P.id}/stories/{S.id}/comments/{C.id}"
+    Then I validate the response contains "text" equals "Comment created by Cucumber"
```

Comment:
Don't forget to delete the projects created in your preconditions.
Apply that approach in all your scenarios.